cosmo_tester/test_suites/test_security/security_test_base.py

```diff
@@ -189,9 +193,13 @@ def _running_env_setup(self):
         self.env.management_ip = self.cfy.get_management_ip()
         self.set_rest_client()
 
-        def clean_mgmt_ip():
+        def clear_mgmt_and_security_settings():
             self.env.management_ip = None
-        self.addCleanup(clean_mgmt_ip)
+            self.env.rest_client = None
+            self.cfy = None
```

Comment:
`cfy` and `client` are overridden by the base class `setUp` for every test so it is not really required cleaning them